tests/core/test_validator.py

```diff
@@ -88,6 +89,29 @@
     )
 
 
+async def test_verify_domain_validity(caplog):
+    domain_path = "data/test_domains/duplicate_intents.yml"
+    full_path = os.path.abspath(domain_path)
+
+    importer = RasaFileImporter(domain_path=domain_path)
+    validator = await Validator.from_importer(importer)
+    validator.verify_domain_validity()
+
+    log_object = caplog.records[-1]
+    message = log_object.message
+    level = log_object.levelname
+
+    assert "WARNING" == level
+    assert (
+        "Loading domain from '{}' failed. Using empty domain. "
+        "Error: 'Intents are not unique! Found two intents with name "
+        "'default'. Either rename or remove one of them.'".format(
+            domain_path, full_path
```

Comment:
Looks like there is only one argument being passed to this `.format()`. Also we're moving to f-strings, that just hasn't been merged to master yet, would you use one here as well?

Reply:
> Functionality looks great  Just some style comments and a revision to the docs after what we changed last time and we're good.
> 
> P.S. sorry to have you waiting and re-merging master, I wasn't aware you were ready (didn't get a notification), in the future feel free to just drop a comment when all the mentioned changes have been integrated and it's ready for another look!

Ok, no problem : )